### ISSUE TITLE
Decorators not showing when file contains emoji

### ISSUE THREAD

**Issue by brollin** [cursorless-dev/cursorless#1038]
For me, under certain conditions when a file contains emoji, the decorators are not shown. They seem to still exist because if I do a command like "take air", it does select something. I ran into this when attempting to "draft all" some text that included emojis. Perhaps that's the best way to attempt to reproduce. Toggling the decorators does not work either.

**Comment by pokey** [comment]
Huh strange. Does the problem go away if you restart VSCode?

**Comment by brollin** [comment]
Nope, I am able to reproduce it consistently. In case it is helpful, this is the page/content that I was "draft all"ing that leads to the issue: https://github.com/brollin/brollin/blob/main/README.md
